reference/configuration/framework.rst

```diff
@@ -427,19 +910,24 @@ Now, the same asset will be rendered as ``/images/logo.png?v2`` If you use
 this feature, you **must** manually increment the ``assets_version`` value
 before each deployment so that the query parameters change.
 
-You can also control how the query string works via the `assets_version_format`_
-option.
+.. tip::
+
+    As with all settings, you can use a parameter as value for the
+    ``assets_version``. This makes it easier to increment the cache on each
+    deployment.
+
+.. _reference-templating-version-format:
 
 assets_version_format
 .....................
 
-**type**: ``string`` **default**: ``%%s?%%s``
+**type**: ``string`` **default**: ``'%%s?%%s'``
 
-This specifies a :phpfunction:`sprintf` pattern that will be used with the `assets_version`_
-option to construct an asset's path. By default, the pattern adds the asset's
-version as a query string. For example, if ``assets_version_format`` is set to
-``%%s?version=%%s`` and ``assets_version`` is set to ``5``, the asset's path
-would be ``/images/logo.png?version=5``.
+This specifies a :phpfunction:`sprintf` pattern that will be used with the
+`assets_version`_ option to construct an asset's path. By default, the pattern
+adds the asset's version as a query string. For example, if
+``assets_version_format`` is set to ``%%s?version=%%s`` and ``assets_version``
+is set to ``5``, the asset's path would be ``/images/logo.png?version=5``.
```

Comment:
They're all correct - it's a possessive apostrophe - the path belongs to the asset :)